src/command/graph/fetch.rs

```diff
@@ -6,18 +6,15 @@ use rover_client::query::schema::get;
 
 use crate::client::get_studio_client;
 use crate::command::RoverStdout;
+use crate::utils::parsers::{parse_graph_ref, GraphRef};
 
 #[derive(Debug, Serialize, StructOpt)]
 pub struct Fetch {
-    /// ID of graph in Apollo Studio to fetch from
-    #[structopt(name = "GRAPH_NAME")]
+    /// <NAME>@<VARIANT> of graph in Apollo Studio to fetch from.
+    /// @<VARIANT> may be left off, defaulting to @current
+    #[structopt(name = "GRAPH_REF", parse(try_from_str = parse_graph_ref))]
```

Comment:
I'm not sure what exactly to call this arg. "Graph Ref" is the technically correct name of this though, and may work??

Reply:
`GRAPH` could also work?? Since technically `graph@variant` describes a single graph 🤔

Reply:
Low effort to rename later on though

Reply:
whatever we refer to this as in the docs should be what we use here